### PR TITLE
dash.ly -> dash (right now pip users get both libraries)

### PR DESCRIPTION
**Pull request by jackwluo**

### CODE REVIEW

**Comment by chriddyp** [comment]
ah thanks @jackwluo ! Can you bump the version too? In the `package.json` and in the `version.py` file. plotly/dash-html-components might need this update too, perhaps plotly/dash-renderer too

File: dash_core_components/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '0.5.0'
+__version__ = '0.6.1'
```
Comment (chriddyp) [inline dash_core_components/version.py]: it should just be `0.5.1` actually

**Comment by jackwluo** [comment]
Fixed

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dash-core-components",
-  "version": "0.5.0",
+  "version": "0.5.2",
```
Comment (chriddyp) [inline package.json]: @jackwluo  - this needs to match the python version. `0.5.1`

**Pull request merged**